docs/source/conf.py

```diff
@@ -64,7 +64,9 @@
 # General information about the project.
 project = "aiida-mlip"
 copyright_first_year = "2024"
-copyright_owners = "Alin M. Elena, Elliot Kasoar, Federica Zanca, Jacob Wilkins"
+copyright_owners = (
+    "Federica Zanca, Elliott Kasoar, Federica Zanca, Jacob Wilkins. Alin M. Elena"
```

Comment:
```suggestion
    "Federica Zanca, Elliott Kasoar, Jacob Wilkins. Alin M. Elena"
```

Reply:
or is the `.` intentional?

Reply:
no mistake also I need to remove one Federica